package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "BarcodeScannerForPhoneGap",
-  "version": "3.1.0",
+  "version": "3.2.1",
```

Comment:
How did you move from v3.1.0 to v3.2.1???? And please check appstore. (Is this a patch or a minor?)

Reply:
There current app store version before the microflow fix is 3.2.0